challenge.c

```diff
@@ -101,7 +120,15 @@ int32_t q4(uint8_t * array, uint32_t arrayLength) {
 // Now, x = 10 and y = 5
 //-------------------------------------------------------------------------
 int q9(int *a, int *b) {
+    if (!a || !b) {
+        return 1;
```

Comment:
At the top of this PR, there's a tab called "Checks". There's a section inside called "Run Tests" which shows the output of all the tests that we run. 
![image](https://user-images.githubusercontent.com/63618806/216846111-1c4dd9f7-b023-4caf-865e-5e242ad18e8e.png)

There were 3 failed tests. Once you get those tests to pass, you can continue with the rest of the onboarding.